### PR TITLE
Add github, linkedin, skills and find command details in DG

### LINKED ISSUE #89: Add details of github, linkedin, skills, findcommand in DG

### PR DESCRIPTION
**Pull request by linustws**
Closes #89 

Trying out this simpler workflow for the first time. (instead of pushing to branch on origin repo)

Added details for github, linkedin and skills.
Added findcommand sequence diagram.
Added some implementation details for findcommand.
Have yet to add activity diagram and some details.
Need to change implementation of find command first.

### CODE REVIEW

File: docs/diagrams/FindSequenceDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+[-> LogicManager : execute("find y/2 m/cs2103t")
+activate LogicManager
+
+LogicManager -> Model : getProtagonist()
+activate Model
+
+Model --> LogicManager : p
+deactivate Model
+
+LogicManager -> CodocParser : parseCommand("find y/2 m/cs2103t, p")
```
[the remark this answers is not in the document]
Comment (linustws) [reply docs/diagrams/FindSequenceDiagram.puml]: Thankss for pointing that out!

File: docs/diagrams/FindSequenceDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+[-> LogicManager : execute("find y/2 m/cs2103t")
+activate LogicManager
+
+LogicManager -> Model : getProtagonist()
+activate Model
+
+Model --> LogicManager : p
+deactivate Model
+
+LogicManager -> CodocParser : parseCommand("find y/2 m/cs2103t", p)
```
Comment (NappySprout) [inline docs/diagrams/FindSequenceDiagram.puml:25]: small typo is updated good work! LGTM

**Review by NappySprout — APPROVED** [review]
LGTM

**Comment by NappySprout** [comment]
LGTM

**Pull request merged**